app/models/application_form.rb

```diff
@@ -34,6 +34,7 @@
 #  received_reference                            :boolean          default(FALSE), not null
 #  reduced_evidence_accepted                     :boolean          default(FALSE), not null
 #  reference                                     :string(31)       not null
+#  references_verified                           :boolean          default(FALSE)
```

Comment:
I think this is better placed on the assessment model actually, to keep things separate we've put most things related to what the assessors do on the assessment.